schemas/user.py

```
@@ -0,0 +1,35 @@
+    name: str
+    password: str = Field(..., min_length=8)
+    nickname: str | None = None
+    profile_image: str | None = None
+
+    @validator('password')
+    def check_password(cls, v):
+        has_upper = any(c.isupper() for c in v)  #대문자 확인
+        has_special = any(c in "!@#$%^&*" for c in v)  #특수 기호 확인
+
+        if not has_upper or not has_special:      #둘중 한개라도 없으면 에러 메세지를 한번에 출력
+            raise ValueError("대문자나 특수기호가 없습니다.")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)